Havit.Blazor.Components.Web.Bootstrap/Offcanvas/HxOffcanvas.razor.cs

```diff
@@ -244,7 +256,28 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
 				{
 					return;
 				}
-				await jsModule.InvokeVoidAsync("show", offcanvasElement, dotnetObjectReference, BackdropEnabledEffective, CloseOnEscapeEffective, ScrollingEnabledEffective);
+				await jsModule.InvokeVoidAsync("show", offcanvasElement, dotnetObjectReference, GetBackdropSetupValue(BackdropEffective), CloseOnEscapeEffective, ScrollingEnabledEffective);
+			}
+		}
+
+		/// <summary>
+		/// Formats a <see cref="OffcanvasBackdrop"/> for offcanvas initialization in JS (it expects a boolean or the string "static").
+		/// </summary>
+		/// <param name="backdrop"></param>
+		/// <returns></returns>
+		private object GetBackdropSetupValue(OffcanvasBackdrop backdrop)
+		{
+			if (backdrop == OffcanvasBackdrop.Static)
+			{
+				return StaticBackdropValue;
+			}
+			else if (backdrop == OffcanvasBackdrop.True)
+			{
+				return true;
+			}
+			else
+			{
+				return false;
```

Comment:
Use the new `data-bs-backdrop="true|false|static"` attribute instead?
(In general we prefer attributes over JavaScript options where possible. It is at least better "readable" when browsing DOM in Developer Tools.)